src/pools/D3MAavePool.sol

```diff
@@ -126,27 +137,26 @@ contract D3MAavePool is ID3MPool {
         emit Rely(usr);
     }
     function deny(address usr) external auth {
+        require(vat.live() == 1, "D3MAavePool/no-deny-during-shutdown");
```

Comment:
why is this needed?

Reply:
You are right, this is not needed as the `hub` uses the other modifier.

Reply:
Removed.